.ci/deps.sh

```diff
@@ -37,9 +37,28 @@ case $CIRCLE_BUILD_IMAGE in
     deps="$deps libxml2-utils php-codesniffer"
     # Add extra infer deps
     deps_infer="$deps_infer ocaml camlp4"
+    # Note that g++ & gfortran 4.9 or higher are also needed for lintr
+    # dependency Rcpp and 5.x is already included due to other dependencies.
+    # However, it needs activating; see immediately below.
     ;;
 esac
 
+# Activate g++ & gfortran 4.9+ for lintr
+ls -al /usr/bin/gcc /usr/bin/g++ /usr/bin/gfortran || true
```

Comment:
I'm busted.  This also caters for Trusty, Yakkety and Zesty ;-)
I'll need to look into why none of these appear in that log.